arviz/stats/stats.py

```diff
@@ -156,6 +156,13 @@ def compare(
     --------
     loo : Compute the Pareto Smoothed importance sampling Leave One Out cross-validation.
     waic : Compute the widely applicable information criterion.
+    plot_compare : Summary plot for model comparison.
+
+    References
+    ----------
+    .. [1] Using Advanced Plot Types in Kaluza: Comparison Plots for Visualization of the TCR V
+        Beta Repertoire, see https://www.beckman.com/flow-cytometry/software/kaluza/learning-center/comparison-plot
```

Comment:
These are called comparison plots too but are actually not related to ours :sweat_smile:. The reference should be https://doi.org/10.1007/s11222-016-9696-4. pylint is also complaining about the line length, I hope the doi link will be shorter, otherwise I don't really know how to fix that, I'd have to search and ask around.

Reply:
Thanks for the review. My bad! Sorry for the confusion. I'll change the reference. Also just to clarify, the references must be linked to a journal?

Reply:
not necessarily, they can also link to books, websites, or even documentation or examples from other libraries. https://arviz-devs.github.io/arviz/api/generated/arviz.plot_loo_pit.html has one of each for example. They will often be papers though, because arviz aims to implement the latest algorithms which are generally not yet covered in books and are generally published as papers that describe them and show they work as intended.